src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeopleTable: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(result => {
+        setPeople(result.map(person => {
+          return {
+            ...person,
+            mother: result.find(mother => mother.name === person.motherName),
+            father: result.find(father => father.name === person.fatherName),
+          };
+        }));
+      })
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  const isPeopleEmpty = !isLoading && !isError && people?.length === 0;
+
+  const { personSlug } = useParams();
+
+  return (
+    <div className="block">
+      <h1 className="title">People Page</h1>
+      <div className="box table-container">
+        {isLoading && (
+          <Loader />)}
+
+        {isError && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {isPeopleEmpty && (
+          <p data-cy="noPeopleMessage">
+            There are no people on the server
+          </p>
+        )}
+
+        {!isLoading && people?.length && (
```

Comment:
You could use the `isPeopleEmpty` here. One less data access // not consequential so you can just keep it this way

Reply:
I'd create a variable for that